examples/tutorials/api/nested_sampling_Crab.py

```diff
@@ -0,0 +1,399 @@
+# Setup
+# -----
+#
+# As usual, we’ll start with some setup …
+#
+
+import matplotlib.pyplot as plt
+import numpy as np
+from gammapy.datasets import Datasets
+from gammapy.datasets import SpectrumDatasetOnOff
+
+from gammapy.modeling.models import (
+    SkyModel,
+    UniformPrior,
+    LogUniformPrior,
+)
+
+from gammapy.modeling.sampler import Sampler
+
+
+######################################################################
+# Loading the spectral datasets
+# -----------------------------
+#
+# Here we will load a few Crab 1D spectral data for which we will do a
+# fit.
+#
+
+path = "$GAMMAPY_DATA/joint-crab/spectra/hess/"
+
+datasets = Datasets()
+for id in ["23526", "23559", "23592"]:
+    dataset = SpectrumDatasetOnOff.read(f"{path}pha_obs{id}.fits")
+    datasets.append(dataset)
+
+
+######################################################################
+# Model definition
+# ----------------
+#
+# Now we want to define the spectral model that will be fitted to the
+# data.
+# The Crab spectra will be fitted here with a simple powerlaw for
+# simplicity.
+#
+
+model = SkyModel.create(spectral_model="pl")
+
+
+######################################################################
+#
+# .. WARNING:: Priors definition:
+#    Unlike a traditional fit where priors on the
+#    parameters is optional, here it is inherent to the Bayesian approach and
+#    are therefore mandatory.
+#
+# In this case we will set (min,max) prior that will define the
+# boundaries in which the sampling will be performed.
+# Note that it is usually recommended to use a `~gammapy.modeling.models.LogUniformPrior` for
+# the parameters that have a large amplitude range like the
+# `amplitude` parameter.
+# A `~gammapy.modeling.models.UniformPrior` means that the samples will be drawn with uniform
+# probability between the (min,max) values in the linear or log space
+# in the case of a `~gammapy.modeling.models.LogUniformPrior`.
+#
+
+model.spectral_model.amplitude.prior = LogUniformPrior(min=1e-12, max=1e-10)
+model.spectral_model.index.prior = UniformPrior(min=1, max=5)
+datasets.models = [model]
+datasets.models
+
+
+######################################################################
+# Defining the sampler and options
+# --------------------------------
+#
+# As for the `Fit` object, the `Sampler` object can receive
+# different backend (although just one is available for now).
+# The `Sampler` comes with “reasonable” default parameters but you can
+# change them via the `sampler_opts` dictionnary.
+# Here is a short description of the most relevant parameters that you
+# could change :
+#
+# -  `live_points`: minimum number of live points throughout the run.
+#    More points allow to discover multiple peaks if existing but is
+#    slower. To test the Prior boundaries and for debugging, a lower
+#    number (~100) can be used before a production run with more points
+#    (~400 or more).
+# -  `frac_remain`: the cut-off condition for the integration. High
```

Comment:
Maybe say what the parameter means as well. `the cut-off condition for the integration, set by the maximum allowed fraction of posterior mass left in the live points vs the dead points` (or similar).